frontend/streamlit_app.py

```diff
@@ -106,6 +145,10 @@ def main() -> None:
                     time.sleep(0.05)
 
                 message_placeholder.markdown(response_buffer)
+            
+            response_time_color = "green" if response_time < 5000 else "orange" if response_time < 10000 else "red"
+            st.markdown(f"<p style='color:{response_time_color};'>Response Time: {response_time / 1000:.2f} seconds</p>", unsafe_allow_html=True)
```

Comment:
why unsafe?

Reply:
<img width="747" alt="image" src="https://github.com/The-OpenROAD-Project/ORAssistant/assets/43810146/b7d366a0-7abe-4261-8cc1-80e789a70958">
without using unsafe.
unsafe is because streamlit sanitize the html added in the markdown, which makes the html not work as expected sometimes, as streamlit can't check on it's own what's inside the html, as people can put javascript in the code, which steamlit can't check, it have unsafe tag.

I found out about inbuilt color for text in streamlit ```
st.markdown('''
    :red[Streamlit] :orange[can] :green[write] :blue[text] :violet[in]
    :gray[pretty] :rainbow[colors] and :blue-background[highlight] text.''')
```

I will replace the previous code without needing unsafe